src/v2/Components/Authentication/Desktop/LoginForm.tsx

```diff
@@ -80,8 +98,11 @@ export class LoginForm extends Component<FormProps, LoginFormState> {
         isLoading: true,
       },
       () => {
-        values.otpRequired = this.state.otpRequired
-        this.props.handleSubmit?.(values, formikBag)
+        const submitValues: SubmitValues = {
```

Comment:
@damassi - Minor, but wanted to revet a change I had made to our previous work on this. Felt it safer to not mutate the values object directly without a better understanding of how this is working under the hood in formik. This now matches our original approach.